common/selectors/transactions.ts

```diff
@@ -28,3 +28,7 @@ export function getRecentWalletTransactions(state: AppState): SavedTransaction[]
     return [];
   }
 }
+
+export function getTransactionToRaw(state: AppState) {
+  return state.transaction.fields.to.raw;
```

Comment:
Also the selectors in this file are for `state.transactions` not `state.transaction`

Reply:
:+1: